src/ir/enum_ty.rs

```diff
@@ -49,6 +54,11 @@ impl Enum {
         &self.variants
     }
 
+    /// The special attributes of enumeration
+    pub fn attributes(&self) -> &Vec<Attribute> {
```

Comment:
Well, the explanation is that `getFieldOffset` returns the offset in bits, according to grepping LLVM's source.